src/renderers/dom/client/utils/DOMChildrenOperations.js

```diff
@@ -247,6 +248,39 @@ var DOMChildrenOperations = {
     }
   },
 
+  startObservation: function() { 
+    var config = { attributes: false, childList: true, characterData: false, subtree: true };
+    var MutationObserver = window.MutationObserver || function() {};
+    MutationObserver.prototype.observe = MutationObserver.prototype.observe || function() {};      
+    MutationObserver.prototype.disconnect = MutationObserver.prototype.disconnect || function() {};      
```

Comment:
I think this should also be a devtool hook as well (see [renderers/dom/shared/hooks](https://github.com/facebook/react/tree/master/src/renderers/dom/shared/hooks)), does that sound right @jimfb?